src/app/Http/Controllers/SettingCrudController.php

```diff
@@ -72,7 +72,7 @@ public function edit($id)
         $this->crud->hasAccessOrFail('update');
 
         $this->data['entry'] = $this->crud->getEntry($id);
-        $this->crud->addField((array) json_decode($this->data['entry']->field)); // <---- this is where it's different
+        $this->crud->addField(json_decode($this->data['entry']->field, true)); // <---- this is where it's different
```

Comment:
The comment's not necessary - git tracks the changes.

That said, this is a subtle behaviour change...it's not likely to break other code but I'm not so sure...

Reply:
Comment was there before me :)

Reply:
Wouldn't it be better to set the second argument to `true`?

(ps. and remove that comment, please)

Reply:
Actually that is what I did (set the second argument to ```true```)